Remote Habits/View/RHDashboardViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+
+import UIKit
+
+class RHDashboardViewController: RHBaseViewController {
+
+    // MARK: - --OUTLETS--
+    @IBOutlet weak var dashboardTableView: UITableView!
+    
+    // MARK: - --VARIABLES--
+    var dashboardData  = UserHabit()
+    var isLoggedIn : Bool = true // Change this by checking logged in user detail using CIO SDK
+    
+    // MARK: - --LIFECYCLE METHODS--
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBar(title: RHConstants.kEmptyValue, hideBack: true, showLogo : true)
+        addDefaultBackground()
+        setupDashboardTableView()
+        dashboardTableView.delegate = self
+        dashboardTableView.dataSource = self
+        // Do any additional setup after loading the view.
+    }
+    
+    // MARK: - --FUNCTIONS--
+    func setupDashboardTableView() {
+        
+        dashboardData = RHStubData().getStubData(isLoggedIn: isLoggedIn)
+        
+        // HabitTableViewCell move to constant file
+        dashboardTableView.register(UINib(nibName: RHConstants.kHabitTableViewCell, bundle: nil), forCellReuseIdentifier: RHConstants.kHabitTableViewCell)
+        dashboardTableView.rowHeight = UITableView.automaticDimension
+        dashboardTableView.estimatedRowHeight = 80
+        
+    }
+    /*
+    // MARK: - --NAVIGATION--
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+    */
+
+}
+
+// MARK: - --UITABLEVIEWDELEGATE--
+extension RHDashboardViewController : UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 97.0
+    }
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let headerView = UIView.init(frame: CGRect.init(x: 0, y: 0, width: tableView.frame.width, height: 97))
+        headerView.backgroundColor = self.view.backgroundColor
+        let headerData = dashboardData[section].first?.key
+        let label = UILabel()
+        label.frame = CGRect.init(x: 16, y: 5, width: headerView.frame.width-10, height: headerView.frame.height-10)
+        if section != 0 {
+            label.frame = CGRect.init(x: 16, y: 60, width: headerView.frame.width-10, height: 26)
+        }
+        
+        label.text = headerData?.headerTitle
+        label.font = UIFont(name: headerData?.titleFontName ?? RHConstants.kEmptyValue, size: CGFloat(headerData?.titleFontSize ?? 17))
+        label.textColor = .black
+        
+        headerView.addSubview(label)
+        
+        return headerView
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        if indexPath.section != 0 { return }
```

Comment:
I found this code take a bit for me to understand what it's doing. Maybe something like this makes it more readable? 

```suggestion
        let isTableViewCell = indexPath.section == 0
        if !isTableViewCell { return }
```